### PR TITLE
Lintron repo permissions

### PR DESCRIPTION
**Pull request by prehnRA**

### CODE REVIEW

File: app/controllers/refresh_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class RefreshController < ApplicationController
```
Comment (lintomat) [inline app/controllers/refresh_controller.rb]: Style/Documentation: Missing top-level class documentation comment.

Comment (lintomat) [inline app/controllers/refresh_controller.rb]: Style/FrozenStringLiteralComment: Missing magic comment `# frozen_string_literal: true`.

File: app/controllers/refresh_controller.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  def create
+    RefreshUserPermissions.run(current_user)
+
+    redirect_back fallback_location: "/"
```
Comment (lintomat) [inline app/controllers/refresh_controller.rb]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: config/routes.rb [hunk 3]
```diff
@@ -10,6 +10,7 @@
   authenticated :user do
     resources :hooks
   end
+  post '/refresh_permissions', to: 'refresh#create', as: "refresh_permissions"
```
Comment (lintomat) [inline config/routes.rb]: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.

**Comment by lintomat** [comment]
To relint, [click here](http://lintron.io/relint/revelrylabs/lintron/307).

**Comment by lintomat** [comment]
To relint, [click here](http://lintron.io/relint/revelrylabs/lintron/307).

**Comment by lintomat** [comment]
To relint, [click here](http://lintron.io/relint/revelrylabs/lintron/307).

**Comment by lintomat** [comment]
To relint, [click here](http://lintron.io/relint/revelrylabs/lintron/307).

File: app/controllers/refresh_controller.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# Allows users to refresh their permissions without logging out and in again
+class RefreshController < ApplicationController
+  def create
+    RefreshUserPermissions.run(current_user)
+
+    redirect_back fallback_location: '/'
```
Comment (grossvogel) [inline app/controllers/refresh_controller.rb]: I wonder if it'd be helpful to show a flash message so the user knows something happened?

**Pull request merged**